src/test/java/org/springframework/data/couchbase/domain/PersonRepository.java

```diff
@@ -0,0 +1,113 @@
+package org.springframework.data.couchbase.domain;
```

Comment:
license headers missing, and commented stuff

Reply:
added license, author, removed spurious comments.